regression-test/suites/datatype_p0/scalar_types/load.groovy

```diff
@@ -90,8 +90,8 @@ suite("test_scalar_types_load", "p0") {
             `c_float` float NULL,
             `c_double` double NULL,
             `c_decimal` decimal(20, 3) NULL,
-            `c_date` date NULL,
-            `c_datetime` datetime NULL,
+            `c_date` datev1 NULL,
+            `c_datetime` datetimev1 NULL,
             `c_datev2` datev2 NULL,
```

Comment:
we should add more test cases including:
1. in list predicate
2. delete predicate
3. comparison predicate
4. is null predicate

test index:
bloom filter index
min max index
bitmap index

test key range

test all data types

test runtime filter // maybe very difficult